internal/template-bundle/bundle_test.go

```diff
@@ -75,6 +87,37 @@ var _ = Describe("Template bundle", Ordered, func() {
 		Expect(ds2.Spec.Source.PVC.Name).To(Equal("win10"))
 		Expect(ds2.Spec.Source.PVC.Namespace).To(Equal("kubevirt-os-images"))
 	})
+
+	It("should throw an error retrieving the bundle file", func() {
+		_, err := RetrieveCommonTemplatesBundleFile(tmpDir)
+		Expect(err.Error()).To(ContainSubstring("failed to find common-templates bundles, none of the files were found"))
+	})
+
+	It("should retrieve the bundle arch independent file", func() {
+		_, err := os.OpenFile(archIndependentFile, os.O_CREATE, 0644)
```

Comment:
Please, close the file.

Reply:
I found a better way. It seems that os.WriteFile implicitly handles both opening and closing the file. Thank you for bringing that to my attention